WooCommerce/Classes/Notifications/LocalNotification.swift

```diff
@@ -128,5 +134,18 @@ extension LocalNotification {
                 comment: "Message on the local notification to remind to continue the Blaze campaign creation."
             )
         }
+        enum ProductImageBackgroundUpload {
+            static let title = NSLocalizedString(
+                "localNotification.ProductImageUploader.title",
+                value: "Product Image Upload in Progress",
+                comment: "Title of the local notification to inform the user that product images are uploading in the background."
+            )
+            static let body = NSLocalizedString(
+                "localNotification.ProductImageUploader.message",
+                value: "Your product images are still uploading, but upload speed may be reduced" +
+                "and errors could occur. For best results, please keep the app open until uploads finish.",
```

Comment:
A space is missing between "reduced" and "and".

I also asked AI for a more concise message, and this is what's suggested:

> Your product images are still uploading in the background. Upload speed may be slower, and errors could occur. For best results, please keep the app open until uploads are complete.